src/main/java/billgates/view/gui/ActionPanel.java

```diff
@@ -223,4 +220,25 @@ private void deleteEntry() {
     public JButton getDeleteEntryButton() {
         return deleteEntryButton;
     }
-}
+
+    // Change color in ActionPanel and statistic text area
+    public void changeColor(Color c){
+        this.setBorder(new CustomTitleBorder("Action", c));
+        this.statisticsTextArea.setBorder(new CustomTitleBorder("Statistics", c));
+        this.statisticsTextArea.setForeground(c);
+    }
```

Comment:
`changeColor` should also change the color of the buttons.

You can simply add `ActionButton.allButton.forEach(b -> b.setBackground(c));` to achieve this.